frontend/assignments/1/home.html

```diff
@@ -0,0 +1,524 @@
+                <div class="tweet-body-main">
+                   <img src="./assets/myassets/img.png" alt="kdu" class="feed-img"/>
+                    <div class="tweet-nav">
+                        <img src="./assets/myassets/comment.png" alt="comment" class="tweet-nav-icon">
+                        <img src="./assets/myassets/retweet.png" alt="retweet" class="tweet-nav-icon">
+                        <img src="./assets/myassets/like.png" alt="like" class="tweet-nav-icon">
+                        <img src="./assets/myassets/insights.png"  alt="insights" class="tweet-nav-icon">
+                        <div class="last-two-icon">
+                            <img src="./assets/myassets/bookmark-dark.png"  alt="bookmark" class="tweet-nav-icon">
+                            <img src="./assets/myassets/save.png"  alt="save" class="tweet-nav-icon">
+                        </div>
+                        
+                    </div>
+                </div>
+              </div>
+            </div>
+
+
+            <div class="tweet">
+                <div class="tweet-img">
+                 <img src="./assets/Profile/profile pic.png" alt="pfp" class="profile-icon"/>
+                </div>
+                <div class="tweet-desc">
+                  <div class="tweet-head">
+                      <div class="tweet-head-info">
+                          <p class="name"> Nitesh Gupta</p>
+                          <p class="username" id="tweet-username">@Nit_hck  <span id="dot"> . </span>  4h</p>
+                      </div>
+                     <p class="more-option"> ...</p>
+                  </div>
+                  <div class="tweet-body-main">
+                  
+                    <video width="300" height="150" controls style="padding-bottom: 4px;">
+                        <source src="movie.mp4" type="video/mp4">
+                        <source src="movie.ogg" type="video/ogg">
+                      </video>
+                   
+                      <div class="tweet-nav">
+                          <img src="./assets/myassets/comment.png" alt="comment" class="tweet-nav-icon">
+                          <img src="./assets/myassets/retweet.png" alt="retweet" class="tweet-nav-icon">
+                          <img src="./assets/myassets/like.png" alt="like" class="tweet-nav-icon">
+                          <img src="./assets/myassets/insights.png"  alt="insights" class="tweet-nav-icon">
+                          <div class="last-two-icon">
+                              <img src="./assets/myassets/bookmark-dark.png"  alt="bookmark" class="tweet-nav-icon">
+                              <img src="./assets/myassets/save.png"  alt="save" class="tweet-nav-icon">
+                          </div>
+                          
+                      </div>
+                  </div>
+                </div>
+              </div>
+            
+            
+        </div>
+        </section>
+        
+
+
+    </div>
+    <script>
```

Comment:
1. Abstract out the script in separate JS file
2. Then, Modularize the code. Break into functions and util files. Arrange accordingly. Use JS docs